src/plugins/saved_objects/public/save_modal/saved_object_save_modal.tsx

```diff
@@ -101,7 +105,10 @@ export class SavedObjectSaveModal extends React.Component<Props, SaveModalState>
             data-test-subj="savedObjectTitle"
             value={title}
             onChange={this.onTitleChange}
-            isInvalid={(!isTitleDuplicateConfirmed && hasTitleDuplicate) || title.length === 0}
+            isInvalid={
```

Comment:
Should we be using the `isInvalid` **and** `error` props on the parent `EuiFormRow` to show some explanatory text as to why the field is considered invalid? Perhaps something like "A title is required"?

Reply:
How about, "You need a title, foo"